src/core_plugins/kbn_vislib_vis_types/public/tile_map.js

```diff
@@ -1,9 +1,7 @@
-import _ from 'lodash';
 import supports from 'ui/utils/supports';
 import MapsVisTypeVislibVisTypeProvider from 'ui/vis_maps/maps_vis_type';
 import VisSchemasProvider from 'ui/vis/schemas';
 import AggResponseGeoJsonGeoJsonProvider from 'ui/agg_response/geo_json/geo_json';
-import FilterBarPushFilterProvider from 'ui/filter_bar/push_filter';
 import tileMapTemplate from 'plugins/kbn_vislib_vis_types/editors/tile_map.html';
 
 export default function TileMapVisType(Private, getAppState, courier, config) {
```

Comment:
Yeah... I think it's the legend, it's just blank. (I'm adding the red boxes to try to highlight what I'm talking about)

<img width="1201" alt="screen shot 2017-03-21 at 4 32 21 pm" src="https://cloud.githubusercontent.com/assets/627123/24169469/19f32a82-0e54-11e7-861a-ee5c329309b3.png">